ext/ddtrace.c

```diff
@@ -1656,6 +1656,32 @@ PHP_FUNCTION(dd_trace_peek_span_id) {
     RETURN_STR(ddtrace_span_id_as_string(ddtrace_peek_span_id()));
 }
 
+/* {{{ proto void dd_trace_close_all_spans_and_flush() */
+PHP_FUNCTION(dd_trace_close_all_spans_and_flush) {
+    UNUSED(execute_data);
+    ddtrace_close_all_spans_and_flush();
+    RETURN_NULL();
+}
+
+/* {{{ proto void dd_trace_synchronous_flush(int) */
+PHP_FUNCTION(dd_trace_synchronous_flush) {
+    zend_long timeout;
+
+    if (zend_parse_parameters_ex(ddtrace_quiet_zpp(), ZEND_NUM_ARGS(), "l", &timeout) == FAILURE) {
+        ddtrace_log_onceerrf("dd_trace_synchronous_flush() expects a timeout in milliseconds");
+        RETURN_NULL();
+    }
```

Comment:
```suggestion
    if (zend_parse_parameters(ZEND_NUM_ARGS(), "l", &timeout) == FAILURE) {
        RETURN_THROWS();
    }
```

Reply:
RETURN_THROWS() just leaves the return value unchanged and asserts an exception happened. For < PHP 8 you probably could just polyfill it in compatibility.h with `#define RETURN_THROWS RETURN_NULL`.